src/NetAppFiles/NetAppFiles/ActiveDirectory/NewNetAppFilesActiveDirectory.cs

```diff
@@ -125,7 +124,13 @@ public class NewAzureRmNetAppFilesActiceDirectory : AzureNetAppFilesCmdletBase
             HelpMessage = "Name of the active directory machine. This optional parameter is used only while creating kerberos volume")]
         [ValidateNotNullOrEmpty]
         public string AdName { get; set; }
-        
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Domain Users in the Active directory to be given Security Privilege (Needed for SMB Continuously available shares for SQL). A list of unique usernames without domain specifier")]
+        [ValidateNotNullOrEmpty]
+        public string[] SecurityOperators { get; set; }
```

Comment:
Please use singular name.

Reply:
Some of those issues PSNetAppFilesVolumeSnapshot and Snapshot  can be problematic as it was incorrectly named and can cause confusion as it is in fact a snapshot **policy** not a snapshot. That is a different type of object even if they are related.

Can not tell why those parameter set issues are flagged can you perhaps elaborate?

Reply:
BTW this confusion with snapshot parameter and snapshotpolicy in New-AzNetAppFilesVolume is leading users to not find the option to set snapshot policy to volumes. Because it is not clear that the snapshot parameter is for setting a snapshot policy.
https://github.com/Azure/azure-powershell/issues/13930

Reply:
@msJinLei we have made some changes, please let me know if the changes are satisfactory.

Reply:
> @msJinLei we have made some changes, please let me know if the changes are satisfactory.

Still two failed test cases. It seems that you need to re-record the failed tests

```
2021-01-26T16:51:18.8573517Z   [xUnit.net 00:00:08.70]     Microsoft.Azure.Commands.NetAppFiles.Test.ScenarioTests.ScenarioTest.BackupTests.TestBackupCrud [FAIL]
2021-01-26T16:51:18.8804227Z     X Microsoft.Azure.Commands.NetAppFiles.Test.ScenarioTests.ScenarioTest.BackupTests.TestBackupCrud [2s 695ms]
2021-01-26T16:51:18.8923702Z EXEC : error Message:  [D:\a\1\s\build.proj]
2021-01-26T16:51:18.8931543Z      System.Management.Automation.ActionPreferenceStopException : The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Unable to find a matching HTTP request for URL 'PUT /subscriptions/69a75bda-882e-44d5-8431-63421204132a/resourceGroups/ps1749/providers/Microsoft.NetApp/netAppAccounts/ps9497?api-version=2020-09-01'. Calling method Item().
2021-01-26T16:51:18.8932753Z     Stack Trace:
2021-01-26T16:51:18.8933578Z        at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
2021-01-26T16:51:18.8934341Z      at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
2021-01-26T16:51:18.8934969Z      at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
2021-01-26T16:51:18.8935624Z      at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2021-01-26T16:51:18.8936358Z      at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2021-01-26T16:51:18.8937095Z      at System.Management.Automation.PowerShell.CoreInvoke[TOutput](IEnumerable input, PSDataCollection`1 output, PSInvocationSettings settings)
2021-01-26T16:51:18.8937738Z      at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings)
2021-01-26T16:51:18.8938244Z      at System.Management.Automation.PowerShell.Invoke()
2021-01-26T16:51:18.8939141Z      at Microsoft.WindowsAzure.Commands.ScenarioTest.EnvironmentSetupHelper.RunPowerShellTest(String[] scripts) in D:\a\1\s\tools\ScenarioTest.ResourceManager\EnvironmentSetupHelper.cs:line 573
2021-01-26T16:51:18.8940273Z      at Microsoft.Azure.Commands.NetAppFiles.Test.ScenarioTests.ScenarioTest.TestController.RunPsTestWorkflow(Func`1 scriptBuilder, Action cleanup, String callingClassType, String mockName) in D:\a\1\s\src\NetAppFiles\NetAppFiles.Test\ScenarioTests\TestController.cs:line 119
2021-01-26T16:51:18.8941985Z      at Microsoft.Azure.Commands.NetAppFiles.Test.ScenarioTests.ScenarioTest.TestController.RunPowerShellTest(XunitTracingInterceptor logger, String[] scripts) in D:\a\1\s\src\NetAppFiles\NetAppFiles.Test\ScenarioTests\TestController.cs:line 68
2021-01-26T16:51:18.8943212Z      at Microsoft.Azure.Commands.NetAppFiles.Test.ScenarioTests.ScenarioTest.BackupTests.TestBackupCrud() in D:\a\1\s\src\NetAppFiles\NetAppFiles.Test\ScenarioTests\BackupTests.cs:line 34
2021-01-26T16:51:18.8944018Z     Standard Output Messages:
```

Reply:
yes having issues on the service side cannot run the tests, it is being looked into hopefully resolved soon.